src/averageAges.js

```diff
@@ -14,14 +14,22 @@
  *
  * @return {number}
  */
-function calculateMenAverageAge(people, century) {
-  // write code here
-  // learn how to use array methods like .filter .map .some .every .find .reduce
-  // avoid using loop and forEach
-  // replace `if ()` statement with &&, || or ?:
-  // without nesting
-}
+const getAverage = function(ages) {
+  const result = ages.reduce(
+    (previousValue, currentValue) => previousValue + currentValue);
+
+  return result / ages.length;
+};
+
+function calculateMenAverageAge(people, century = 0) {
+  const arrayMen = people.filter(
```

Comment:
Don't use data type names in variable names
```suggestion
  const men = people.filter(
```